packages/ilmomasiina-backend/src/routes/auditlog/getAuditLogs.ts

```diff
@@ -0,0 +1,71 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { Op, WhereOptions } from 'sequelize';
+
+import * as schema from '@tietokilta/ilmomasiina-models/src/schema';
+import { AuditLog } from '../../models/auditlog';
+import { stringifyDates } from '../utils';
+
+const MAX_LOGS = 100;
+
+export default async function getAuditLogItems(
+  request: FastifyRequest<{ Querystring: schema.AuditLoqQuery }>,
+  response: FastifyReply,
+): Promise<schema.AuditLogResponse> {
+  let where: WhereOptions<AuditLog>[] = [];
```

Comment:
For whatever reason this had broken for me. Don't know how it worked before.
```suggestion
  let where: WhereOptions<AuditLog> = [];
```

Reply:
Should work without modifications if `action` query param is defined to be an array.

Reply:
Seems that this is something that's not detected by TypeScript 4.3 (which I pinned in `package.json` due to a previous issue) but is detected by TypeScript 4.8 (which VS Code uses by default). We should probably fix it to allow upgrading TypeScript later.

Reply:
Resolved in 199fa35